README.md

```diff
@@ -85,7 +85,10 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/Ant-Shell"><img src="https://avatars.githubusercontent.com/u/100455148?v=4?s=100" width="100px;" alt="Anthony Shellman"/><br /><sub><b>Anthony Shellman</b></sub></a><br /><a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=Ant-Shell" title="Code">💻</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=Ant-Shell" title="Documentation">📖</a> <a href="#research-Ant-Shell" title="Research">🔬</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/alanoteles"><img src="https://avatars.githubusercontent.com/u/1028089?v=4?s=100" width="100px;" alt="Alano Teles"/><br /><sub><b>Alano Teles</b></sub></a><br /><a href="#question-alanoteles" title="Answering Questions">💬</a> <a href="#blog-alanoteles" title="Blogposts">📝</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=alanoteles" title="Code">💻</a> <a href="#content-alanoteles" title="Content">🖋</a> <a href="#data-alanoteles" title="Data">🔣</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=alanoteles" title="Documentation">📖</a> <a href="#research-alanoteles" title="Research">🔬</a> <a href="#translation-alanoteles" title="Translation">🌍</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/ht-l1"><img src="https://avatars.githubusercontent.com/u/106502799?v=4?s=100" width="100px;" alt="Hannah L"/><br /><sub><b>Hannah L</b></sub></a><br /><a href="#question-ht-l1" title="Answering Questions">💬</a> <a href="https://github.com/TejasQ/opensauced-guestbook/issues?q=author%3Aht-l1" title="Bug reports">🐛</a> <a href="#business-ht-l1" title="Business development">💼</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=ht-l1" title="Code">💻</a> <a href="#ideas-ht-l1" title="Ideas, Planning, & Feedback">🤔</a> <a href="#infra-ht-l1" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a> <a href="#maintenance-ht-l1" title="Maintenance">🚧</a></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/ht-l1"><img src="https://avatars.githubusercontent.com/u/106502799?v=4?s=100" width="100px;" alt="Hannah Lin"/><br /><sub><b>Hannah Lin</b></sub></a><br /><a href="#question-ht-l1" title="Answering Questions">💬</a> <a href="https://github.com/TejasQ/opensauced-guestbook/issues?q=author%3Aht-l1" title="Bug reports">🐛</a> <a href="#business-ht-l1" title="Business development">💼</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=ht-l1" title="Code">💻</a> <a href="#ideas-ht-l1" title="Ideas, Planning, & Feedback">🤔</a> <a href="#infra-ht-l1" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a> <a href="#maintenance-ht-l1" title="Maintenance">🚧</a></td>
```

Comment:
Here's where Hannah's info is changed.

Reply:
That is so weird I didn't modify anyone's info. I'll fix that though, thanks!

Reply:
I'm thinking this could be GitHub co pilot or some type of formatter I have installed because I've noticed a change happen like this in one of my other projects...